doc/specs/#980 - SnapOnOutput.md

```diff
@@ -0,0 +1,100 @@
+---
+author: Carlos Zamora @carlos-zamora
+created on: 2019-08-22
+last updated: 2020-05-13
+issue id: 980
+---
+
+# Snap On Output
+
+## Abstract
+
+The goal of this change is to introduce a new profile-level setting `snapOnOutput` that allows users to control the Terminal's scroll response to newly generated output.
+
+Currently, new output causes the Terminal to always scroll to it. Some users want to be able to scroll through the buffer without interruptions. This setting allows the Terminal to conditionally or never scroll to the new output.
+
+## Inspiration
+
+Creating a selection when the terminal is generating more output is difficult. This is because the terminal scrolls to the new output.
+
+In CMD, a selection causes any processes to run in the background, but prevents more output from being displayed. When the selection is removed, the output is flushed to the display.

```

Comment:
In conhost.

... causes the active process to be paused ...

completely paused.